source/inverse.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Функция возвращает инвертированный массив, если он без дополнительного параметра.
+ * Если передать положительный параметр n вместе с массивом, 
+ * он инвертирует массив с n-го элемента, с отрицательным -до |n|-го элемента.
+ * @param {array} array инвертируемый массив
+ * @param {number} index индекс, от(до) которого идет инверсия
+ * @returns {array} возвращаемый массив
+ */
+const inverse = (numbers, index = 0) => {
+    if (!Array.isArray(numbers)) {
+        throw new TypeError('Parameter is not an array!');
```

Comment:
```suggestion
const inverse = (array, index = 0) => {
    if (!Array.isArray(array)) {
        throw new TypeError('Parameter is not an array!');
```
ахахах - я что-то подумал - а почему "входной массив с абстрактными элементами" называется numbers?) давай его в array переименуем =)